config/deploy.secret.exs

```diff
@@ -0,0 +1,10 @@
+use Mix.Config
+
+# Configure your database
```

Comment:
it seems to me that `mix compile` is not building the _values_ of the env variables into the container, but rather, it is building the instruction to read from those variables into the container.  this is the desired behaviour for a docker build, and the `deploy.secret.exs` is not used in any other context, so i don't think there is a problem.

i have also already updated the build/deploy section to include instructions about writing the `prod.secret.exs` file before running in production mode, so i don't think i need to do anything further there.